prism-agent/service/server/src/main/scala/io/iohk/atala/verification/controller/http/VcVerification.scala

```diff
@@ -0,0 +1,65 @@
+package io.iohk.atala.verification.controller.http
+
+import io.iohk.atala.pollux.core.service
+import io.iohk.atala.pollux.core.service.verification.VcVerification as ServiceVcVerification
+import sttp.tapir.Schema
+import zio.json.{DeriveJsonDecoder, DeriveJsonEncoder, JsonDecoder, JsonEncoder}
+
+enum VcVerification {
+  case SignatureVerification
+  case IssuerIdentification
+  case ExpirationCheck
+  case NotBeforeCheck
+  case AudienceCheck
+  case SubjectVerification
+  case IntegrityOfClaims
+  case ComplianceWithStandards
+  case RevocationCheck
+  case AlgorithmVerification
+  case SchemaCheck
+  case SemanticCheckOfClaims
+}
+
+object VcVerification {
```

Comment:
It's up to you @CryptoKnightIOG, but if you want to reduce the conversion logic, you can also expose the `VcVerification` from the service layer right away. It seems they have the exact same structure, right? We do that a lot in other parts of the codebase...

Reply:
Will fix in follow up